server/src/main.js

```diff
@@ -120,6 +125,55 @@ app.use('/hook', bodyParser.json(), getCallback('ses-hook', creators, models, pu
 
 const graphQLServer = createServer(app);
 
+// eslint-disable-next-line no-unused-vars
+const subscriptionServer = SubscriptionServer.create({
+  schema,
+  execute,
+  subscribe,
+  onConnect(connectionParams) {
+    let user;
+    let device;
+
+    if (connectionParams.jwt) {
+      jsonwebtoken.verify(connectionParams.jwt, JWT_SECRET,
```

Comment:
this should definitely 100% be abstracted out to a separate module rather than copy paste from our other auth verification methods. Someone will change one and not the other one day and it will break the world.